dockers/docker-snmp/snmp_yml_to_configdb.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import yaml
+from swsssdk import ConfigDBConnector
+
+db = ConfigDBConnector()
+db.connect()
+
+with open('/etc/sonic/snmp.yml', 'r') as yaml_file:
```

Comment:
>yaml_file [](start = 41, length = 9)

If this file is removed in future, crash?

Reply:
So we will be able to remove this in the future.  There is another PR required for that but this PR is a non-intrusive first step.

Reply:
Just in case, you could add a check that the file exists first (or surround the entire block in a `try/except` statement) to fail gracefully if the file doesn't exist.

Reply:
Spoke with Qi last night and clarified what he was asking.  I've updated this to verify if the file exists or not. 

Here is the test I did after my update: 

```
admin@str-s6000-acs-11:~$ redis-cli -n 4 keys "*" | grep -i snmp 
SNMP_COMMUNITY|Joker
SNMP_COMMUNITY|public
FEATURE|snmp
SNMP_COMMUNITY|Kat
SNMP|CONTACT
SNMP|LOCATION
SNMP_COMMUNITY|Travis
admin@str-s6000-acs-11:~$ redis-cli -n 4 hgetall "SNMP_COMMUNITY|public"
1) "TYPE"
2) "RO"
admin@str-s6000-acs-11:~$ 
admin@str-s6000-acs-11:~$ 
admin@str-s6000-acs-11:~$ 
admin@str-s6000-acs-11:~$ 
admin@str-s6000-acs-11:~$ sudo rm /etc/sonic/snmp.yml
admin@str-s6000-acs-11:~$ cat /etc/sonic/snmp.yml
cat: /etc/sonic/snmp.yml: No such file or directory
admin@str-s6000-acs-11:~$ 
admin@str-s6000-acs-11:~$ 
admin@str-s6000-acs-11:~$ docker stop snmp
snmp
admin@str-s6000-acs-11:~$ docker start snmp
snmp
admin@str-s6000-acs-11:~$ redis-cli -n 4 keys "*" | grep -i snmp 
SNMP_COMMUNITY|Joker
SNMP_COMMUNITY|public
FEATURE|snmp
SNMP_COMMUNITY|Kat
SNMP|CONTACT
SNMP|LOCATION
SNMP_COMMUNITY|Travis
admin@str-s6000-acs-11:~$ 
```